rootfs/standard/var/pynode/application_info.py

```diff
@@ -738,6 +754,39 @@ def restart_application(short_name):
     except Exception as e:
         return False
 
+def backup_data_folder(app_data):
+    log_message("  Running backup_data_folder...")
```

Comment:
It looks like backup and restore are still unimplemented.

Reply:
see https://github.com/mynodebtc/mynode/pull/940#discussion_r2158707230